src/plugins/telemetryTable/collections/SortedTableRowCollection.js

```diff
@@ -162,6 +167,19 @@ define(
                 }
             }
 
+            sortByTimeSystem(timeSystem) {
```

Comment:
I think this may have been from having multiple TC's at once, those are sorted, but all are not sorted together. I've removed all this stuff now and created one TableRowCollection that houses the telemetry collections as well so it has access to ALL bounded telemetry.